tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AspNetCore/DefaultModelBindingContext_SetResult_Integration.cs

```diff
@@ -58,43 +60,46 @@ internal static CallTargetReturn OnMethodEnd<TTarget>(TTarget instance, System.E
                 {
```

Comment:
As we've enabled nullable reference types, we should probably also change the method signature to be `System.Exception? exception` to ensure it's correct

Reply:
Good point. I have added the nullable.